runtime/polkadot/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("polkadot"),
 	impl_name: create_runtime_str!("parity-polkadot"),
 	authoring_version: 0,
-	spec_version: 9180,
```

Comment:
I have a migration in the same file that executes conditionally based on 9180 (not a good way, won't do it again). Please change that to make sure it will be executed.

Reply:
assuming that there will be a separate 9181 release for polkadot, which I inferred based on your spec bump.

Reply:
yeah, seems not a great approach for migrations. i think we should prefer temporary things in storage